morse.rb

```diff
@@ -0,0 +1,57 @@
+MORSE = {
+  'a' => '.-',
+  'b' => '-...',
+  'c' => '-.-.',
+  'd' => '-..',
+  'e' => '.',
+  'f' => '..-.',
+  'g' => '--.',
+  'h' => '....',
+  'i' => '..',
+  'j' => '.---',
+  'k' => '-.-',
+  'l' => '.-..',
+  'm' => '--',
+  'n' => '-.',
+  'o' => '---',
+  'p' => '.--.',
+  'q' => '--.-',
+  'r' => '.-.',
+  's' => '...',
+  't' => '-',
+  'u' => '..-',
+  'v' => '...-',
+  'w' => '.--',
+  'x' => '-..-',
+  'y' => '-.--',
+  'z' => '--..',
+  '0' => '-----',
+  '1' => '.----',
+  '2' => '..---',
+  '3' => '...--',
+  '4' => '....-',
+  '5' => '.....',
+  '6' => '-....',
+  '7' => '--...',
+  '8' => '---..',
+  '9' => '----.',
+  ' ' => ' '
+}.freeze
+
+def decode_char(code)
+  MORSE.key(code).upcase
+end
+
+def decode_word(word)
+  text = []
+  word.split(/ /).each { |code| text.push(decode_char(code)) }
+  text.join
+end
+
+def decode_sentence(sentence)
+  text = []
+  sentence.split(/ /).each { |word| text.push(decode_word(word)) }
+  text.join(' ')
+end
+
+puts decode_sentence '.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...'
```

Comment:
When printing sentence there is an extra space between characters. Kindly, fix that spacing to be readable sentence like this `A BOX FULL OF RUBIES`

Your print:

<img width="794" alt="image" src="https://user-images.githubusercontent.com/34578631/179019450-80167095-04be-48cd-8350-5557cd3b5aff.png">